tsconfig.json

```diff
@@ -0,0 +1,106 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Projects */
```

Comment:
Is it a good practice to keep all these comments ?

Reply:
I think so. It's more a matter of personal preference. We could remove them but it's easier later on if we wanna edit the file to have all the available options.